client/packages/common/src/hooks/useUrlQuery/useUrlQuery.ts

```diff
@@ -16,17 +16,14 @@ export const useUrlQuery = ({ skipParse = [] }: useUrlQueryProps = {}) => {
   const updateQuery = (values: UrlQueryObject, overwrite = false) => {
     const newQueryObject = overwrite
       ? {}
-      : { ...parseSearchParams(searchParams, skipParse) };
+      : Object.fromEntries(searchParams.entries());
```

Comment:
a quick tidy while I'm here. No need to create a parsed object and then convert all values to strings again.

Reply:
that also looks much better thanks